CLI fails with odd error warnings

## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
OK, the npm script is "almost" working now, but getting these strange errors
```shell
$ npm run build

> greenwood-getting-started@1.0.0 build /Users/owenbuckley/Workspace/github/repos/greenwood-getting-started
> greenwood build

/Users/owenbuckley/Workspace/github/repos/greenwood-getting-started/node_modules/.bin/greenwood: line 1: /Applications: is a directory
/Users/owenbuckley/Workspace/github/repos/greenwood-getting-started/node_modules/.bin/greenwood: line 3: //: is a directory
/Users/owenbuckley/Workspace/github/repos/greenwood-getting-started/node_modules/.bin/greenwood: line 4: syntax error near unexpected token `0'
/Users/owenbuckley/Workspace/github/repos/greenwood-getting-started/node_modules/.bin/greenwood: line 4: `process.setMaxListeners(0);'
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! greenwood-getting-started@1.0.0 build: `greenwood build`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the greenwood-getting-started@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/owenbuckley/.npm/_logs/2019-08-12T23_58_29_335Z-debug.log
```

## Details
Looked at other scripts in _.bin/_ and looks like we are missing this at the top our _index.js_ file
```shell
#!/usr/bin/env node
```

With that in place manually..... 🥁 
```shell
$ npm run build

> greenwood-getting-started@1.0.0 build /Users/owenbuckley/Workspace/github/repos/greenwood-getting-started
> greenwood build

-------------------------------------------------------
Welcome to Greenwood ♻️
-------------------------------------------------------
Reading project config
Initializing project workspace contexts
Generating graph of workspace files...
Scaffolding out project files...
Generate pages from templates...
Writing imports for md...
Writing Lit routes...
setup index page and html
Scaffolding complete.
Building project for production.
Building SPA from compilation...
webpack build complete
...................................
Static site generation complete!
...................................
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@greenwood/cli",
-  "version": "0.3.1",
+  "version": "0.3.0",
   "description": "A modern and performant static site generator supporting Web Component based development.",
   "repository": "git@github.com:ProjectEvergreen/greenwood.git",
   "author": "Owen Buckley <owen@thegreenhouse.io>",

